source/Playnite/WindowsNotifyIconManager.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Playnite
+{
+    public static class WindowsNotifyIconManager
+    {
+        public static void Notify(Icon icon, string title, string body, Action clickAction = null)
+        {
+            var notifyIcon = new NotifyIcon
+            {
+                Icon = icon,
+                BalloonTipTitle = title,
+                BalloonTipText = body,
+                Visible = true
+            };
+            notifyIcon.BalloonTipClicked += (o, ea) => { clickAction?.Invoke(); notifyIcon.Dispose(); };
+            notifyIcon.BalloonTipClosed += (o, ea) => { notifyIcon.Dispose(); };
```

Comment:
This line I'm not exactly sure about. If we dispose of the icon, it properly disappears from the Windows taskbar. However, the W10 notification disappears along with it, so there's no chance the user will be able to see it after a gaming session.

If we don't dispose of the icon, it stays in the taskbar with no chance of cleaning it up later (even the click action stops working after the 5sec timeout)